tests/test_meshtastic_utils.py

```
@@ -2339,6 +2411,40 @@ def test_seed_connect_time_skew_logs_post_start_calibration(self):
+        mu._relay_connection_started_monotonic_secs = 1_000.0
+        mu.RELAY_START_TIME = 100_000.0
+        mu._relay_startup_drain_deadline_monotonic_secs = None
+        mu._relay_reconnect_prestart_bootstrap_deadline_monotonic_secs = 1_005.0
+
+        with (
+            patch("mmrelay.meshtastic_utils.time.time", return_value=100_000.0),
+            patch("mmrelay.meshtastic_utils.time.monotonic", return_value=1_001.0),
+        ):
+            first_result = mu._seed_connect_time_skew(94_900.0)
+
+        self.assertTrue(first_result)
```

Ruff (0.15.9)

[warning] Use a regular `assert` instead of unittest-style `assertTrue`

Replace `assertTrue(...)` with `assert ...`

(PT009)